docs/userGuide/contentAuthoring.md

```diff
@@ -284,6 +285,83 @@ Front matter can also be included from a separate file, just like how content ca
 
 This will result in `index.md` having the title `Binary Search Tree` and the specified keywords in order for it to be looked up through the search bar.
 
+### Site Navigation
+
+A site navigation bar is a fixed menu on the left side of your page contents, that allows the viewer to navigate your site pages. 
+It has a fixed width of 300 pixels for its contents. 
+If you would like a site navigation bar for your site, you may create one easily using Markdown.
+
+- Start by creating a markdown file (`.md`) in `_markbind/navigation`.
+    - All new sites created with `init` have a `site-nav.md` file created automatically for you as a template in `_markbind/navigation`.
+    - More than one navigation file can be created for different pages.
+
+- Author your navigation layout using Markdown's unordered list syntax as shown below.
+    - When you are using links (`[linkName](url)`), ensure that the url starts from the root directory `/` when you are referencing a page in your site. e.g. `/folder1/myPage.html`
+
+```html
+<!-- In _markbind/navigation/mySiteNav.md -->
+<markdown>
+- [Home :house:](/index.html)
+- Dropdown title :pencil2: <!-- Nested list items will be placed inside a Dropdown menu -->
+  - Dropdown item one
+  - Nested Dropdown title :triangular_ruler:
+    - Nested Dropdown item one
+    - Nested Dropdown item two
+  - Dropdown item two

```

Comment:
Why do we have items without links here?

Reply:
It serves as a working example of how the page navigation is rendered.  Much like how the default content for footers have `This is a dynamic height footer ..... ` for the Author to edit out after seeing how it is rendered.

Reply:
hmm... it is a bit misleading to give examples like this one as this one will not work as expected (due to missing links).